packages/react-components/.storybook/preview.js

```diff
@@ -0,0 +1,26 @@
+import * as rootPreview from '../../../.storybook/preview';
+
+// load global styles
+import '../public/intro.css';
+
+/** @type {typeof rootPreview.decorators} */
+export const decorators = [...rootPreview.decorators];
+
+/** @type {typeof rootPreview.parameters} */
+export const parameters = {
+  ...rootPreview.parameters,
+  options: {
+    storySort: {
+      /**
+       * @see https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories
+       */
+      order: [
```

Comment:
partially copied from `react-examples/.storybook` and used latest SB syntax for "ordering"